### PR TITLE
PIB - Implement I belong copy changes across site

### PR DESCRIPTION
**Pull request by A-Wheeto**
## Status

* Current Status: Ready for (front-end / tech) review
* Closes https://github.com/NCCE/teachcomputing.org-issues/issues/2810

## Review progress:

- [ ] Browser tested
- [ ] Front-end review completed
- [ ] Tech review completed

## What's changed?
- Added I belong banner to primary-teachers and primary-senior-leaders page
- Copy changes on various pages, banners and cards in line with specification document

### CODE REVIEW

File: app/components/course_grid_component/course_grid_component.html.erb [hunk 1]
```diff
@@ -49,10 +49,10 @@
           </div>
       </div>
     </div>
-    <p class="govuk-body govuk-!-text-align-centre">
+    <p class="govuk-body govuk-!-text-align-centre govuk-!-margin-bottom-0">
```
Comment (A-Wheeto) [inline app/components/course_grid_component/course_grid_component.html.erb:52]: Spacing was a bit too much at the bottom of the course component - this is to reduce it slightly and look neater.

File: app/views/certificates/i_belong/complete.html.erb [hunk 2]
```diff
@@ -48,10 +48,16 @@
...
+          This is just the start of your journey!
+        </p>
+        <p class="govuk-body">
+          Now you’ve started implementing strategies to support girls into computer science from primary, during the transition, in secondary and beyond, it’s important to continue doing so, embedding and evaluation as you go along. Evidence your great work through the
           <%= link_to 'Computing Quality Framework.', 'https://computingqualityframework.org/' %>
         </p>
-        <p class="govuk-body">Help to widen impact through your network by sharing your knowledge and practice. If you haven’t already, don’t forget to claim your I Belong champion digital badge and check the <%= link_to 'information pack', 'https://static.teachcomputing.org/IBelong_InfoPack.pdf' %> for ideas and support on how you can do this.</p>
+        <p class="govuk-body">
+          Help to widen impact through your network by sharing your knowledge and practice – you can check out the I Belong Champion <%= link_to 'information pack', 'https://static.teachcomputing.org/IBelong_InfoPack.pdf' %> for ideas and support on how you can do this.
```
Comment (msquance-stem) [inline app/views/certificates/i_belong/complete.html.erb:58]: Should we move the Info Pack link to a external link helper method?

And do the computing quality framework one at the same time?

**Review by msquance-stem — APPROVED** [review]
LGTM

I had added a point about replacing a couple of hard linked urls in one of the files you edited. Not essential, but seeing as we are already modifying the file, might be worth it?

File: app/views/certificates/i_belong/complete.html.erb (hunk 2 above)
Comment (A-Wheeto) [reply app/views/certificates/i_belong/complete.html.erb:58]: Not sure why I didn't do this to start with, we already have a helper method for CQF. I will move the info pack too.

**Pull request merged**